exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/api/ConnectionApi.kt

```diff
@@ -6,6 +6,7 @@ interface ExposedConnection<OriginalConnection : Any> {
     fun rollback()
     fun close()
     var autoCommit: Boolean
+    var isReadOnly: Boolean
```

Comment:
Maybe it's for to rename to `readOnly` as the value can be set here?

Reply:
renamed to `readOnly`